samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java

```diff
@@ -128,18 +129,16 @@ public MultiLingualBot(UserState withUserState) {
     }
 
     private static CompletableFuture<Void> sendWelcomeMessage(TurnContext turnContext) {
-        for (ChannelAccount member: turnContext.getActivity().getMembersAdded()) {
-            // Greet anyone that was not the target (recipient) of this message.
-            // To learn more about Adaptive Cards, see https://aka.ms/msbot-adaptivecards for more details.
-            if (member.getId() != turnContext.getActivity().getRecipient().getId()) {
+        return turnContext.getActivity().getMembersAdded().stream()
+            .filter(member -> !StringUtils.equals(member.getId(), turnContext.getActivity().getRecipient().getId()))
```

Comment:
We can create a variable to replace `turnContext.getActivity().getRecipient().getId()` on this line and make it more readable

Reply:
Agree with you, but we are not re-using the `turnContext.getActivity().getRecipient().getId()` again apart from that line. In other case, I'd like you approach, but not here